publish_pipeline/generate_high_level_metadata/generate_readme.py

```diff
@@ -121,11 +112,45 @@ def pipeline():
 
     readme_metadata["StandardsFollowed"] = standards_followed
 
-    # todo: resources
+    # resources
+    resource = ""
+    cur.execute(
+        "SELECT resource_type FROM dataset_other WHERE dataset_id = %s",
+        (dataset_id,),
+    )
+
+    dataset_other_resource = cur.fetchone()
+    # resource = dataset_other.join(",")
+    resource = dataset_other_resource[0]
+    readme_metadata["Resource"] = resource
 
-    # todo: license
+    # license
 
-    # todo: create citation
+    license_text = ""
+
+    cur.execute(
+        "SELECT rights FROM dataset_rights WHERE dataset_id = %s",
+        (dataset_id,),
+    )
+
+    dataset_rights = cur.fetchone()
+    # license_text = dataset_other.join(",")
+    license_text = dataset_rights[0]
+    readme_metadata["License"] = license_text
```

Comment:
This should be the license identifier here. Check the library to see what it uses

Reply:
in DB we dont have any licence identifier, only license text exists. In that case what represent that license id?

Reply:
dataset rights table has the identifer. rights are licenses. The table will only have one per dataset as per the frontend UI